docs/source/getting_started.md

```diff
@@ -95,6 +95,9 @@ For that, a 3 layer schema is used:
 The closer to the user, the closer to the ontology concepts.
 The abstraction is replaced by specificity when you move towards the backend.
 
+For example, the City, Street or Neighborhood classes from the demonstrative [City Ontology](./ontologies_included.html#the-city-ontology) included in OSP-core, as well as the individuals that can be instantiated using them, would be part of the semantic layer. Any wrapper (i.e. the included [SQLite wrapper](https://github.com/simphony/osp-core/tree/master/osp/wrappers/sqlite)), would be part of the interoperability layer. Finally, following the SQLite example, the [sqlite3 library](https://docs.python.org/3/library/sqlite3.html) from python would be part of the syntactic layer.
```

Comment:
_e.g._ fits better, rather than _i.e._ when mentioning the SQLite wrapper

Reply:
Thanks! Did not know _exempli gratia_. Changed in [c18ae2b](https://github.com/simphony/docs/pull/111/commits/c18ae2b9f631436a8383fdf2a8813f285e321902).